v0/destinations/facebook_pixel/transform.js

```diff
@@ -23,8 +23,10 @@ const {
   getIntegrationsObj,
   getSuccessRespEvents,
   isObject,
-  getValidDynamicFormConfig
+  getValidDynamicFormConfig,
+  isDefinedAndNotNull
 } = require("../../util");
+const { isDefined } = require("rudder-transformer-cdk/build/utils");
```

Comment:
We don't need to import `isDefined` from build, I think. Can you once try without `build` @shrouti1507  ?

Reply:
yes, did it by mistake. Removed it.